### PR TITLE
Code quality

### PR DESCRIPTION
**Pull request by mcg-web**
Adding some tools to help maintainers.

### CODE REVIEW

**Comment by vladar** [comment]
Seet, thanks! But can you please change the target branch from 0.12.x to master? I don't think we'll do much work in 0.12.x branch anymore (just minor fixes).

File: composer.json [hunk 1]
```diff
@@ -18,9 +18,6 @@
   },
   "config": {
     "bin-dir": "bin",
-    "platform": {
```
Comment (mcg-web) [inline composer.json]: Remove this to test lib in environment without faking platform.

**Comment by mcg-web** [comment]
@vladar 0.12 is break for php 7.3 nightly and platform config was hiding this

**Comment by vladar** [comment]
Yeah, but those fixes will stay in 0.12.x, or do you plan to merge them later to master?

**Comment by mcg-web** [comment]
0.12.x should always be merged in master, this is the best way to maintain both branch while 0.12 is still maintained, yes I will merged it in master.

File: composer.json [hunk 2]
```diff
@@ -36,6 +34,15 @@
       "GraphQL\\Examples\\Blog\\": "examples/01-blog/Blog/"
     }
   },
+  "scripts": {
+    "static-analysis": [
+      "rm phpstan.phar || true",
```
Comment (simPod) [inline composer.json]: why not simply use phpstan as a dev dependency? this looks unnecessarily complicated

Comment (mcg-web) [reply composer.json]: cause phpstan require php 7.1 ;)

Comment (simPod) [reply composer.json]: This lib does [too](https://github.com/webonyx/graphql-php/blob/master/composer.json#L12), hm 🤔?

Comment (mcg-web) [reply composer.json]: I m not on master ;) but 0.12.x

Comment (simPod) [reply composer.json]: Not sure if I understand correctly :D This is how it should look like 

https://github.com/doctrine/dbal/blob/master/composer.json#L23

And in travis, simply use `script: vendor/bin/phpstan analyse`

**Comment by mcg-web** [comment]
@vladar can you please review this and tell me if I need to split this in two PR. I need to fix nightly build for the bundle ;)

**Comment by mcg-web** [comment]
Thank you I'm working on a PR to merge these changes in master?

**Pull request merged**